powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/controller/SecureVaultController.java

```diff
@@ -49,6 +52,9 @@
 
     private PowerAuthServiceClient powerAuthClient;
 
+    @Autowired
```

Comment:
Could we use `HttpServletRequest` as method parameter, instead of autowired property? While this is in fact equivalent ([while not obvious](https://stackoverflow.com/questions/48574780/autowired-httpservletrequest-vs-passing-as-parameter-best-practice)), we will avoid autowiring properties (produces warning)...